jbpm-server-distribution/pom.xml

```diff
@@ -3,13 +3,18 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.kie</groupId>
-    <artifactId>kie-server-distributions</artifactId>
+    <artifactId>kie-wb-distributions</artifactId>
     <version>7.15.0-SNAPSHOT</version>
   </parent>
   <artifactId>jbpm-server-distribution</artifactId>
   <packaging>pom</packaging>
   <name>jBPM server distribution</name>
   <description>Single ZIP jBPM server distribution</description>
+
+  <properties>
+    <version.org.postgresql>42.2.1</version.org.postgresql>
+    <version.org.wildfly>14.0.1.Final</version.org.wildfly>
```

Comment:
Please remove the `version.org.wildfly` from here. It's already declared in kie-parent [here](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L261)